### PR TITLE
feat: if preselected empty we are using default list

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
Providing user with preselected default list mime types for upload interaction.

https://oat-sa.atlassian.net/browse/AUT-19

### CODE REVIEW

File: views/js/qtiCreator/widgets/interactions/uploadInteraction/states/Question.js [hunk 1]
```diff
@@ -40,6 +41,13 @@ define([
             // Pre-select a value in the types combo box if needed.
             preselected = uploadHelper.getExpectedTypes(interaction);
 
+        const config = module.config();
```
Comment (mike-ionut-mihai-sandu-tao) [inline views/js/qtiCreator/widgets/interactions/uploadInteraction/states/Question.js:44]: i think you need to run 
```
npx prettier --write and the location of this file
npx prettier --check
npm eslint
```

File: views/js/qtiCreator/widgets/interactions/uploadInteraction/states/Question.js [hunk 2]
```diff
@@ -40,6 +41,13 @@ define([
...
+
+        if (preselected.length === 0 && config.defaultList.length > 0) {
```
Comment (bziondik) [inline views/js/qtiCreator/widgets/interactions/uploadInteraction/states/Question.js:46]:
```suggestion
        if (preselected.length === 0 && config.defaultList && config.defaultList.length > 0) {
```

File: views/js/qtiCreator/widgets/interactions/uploadInteraction/states/Question.js [hunk 3]
```diff
@@ -40,6 +41,13 @@ define([
...
+            preselected = preselected.concat(config.defaultList)
```
Comment (bziondik) [inline views/js/qtiCreator/widgets/interactions/uploadInteraction/states/Question.js:47]:
```suggestion
            preselected = preselected.concat(config.defaultList);
```

**Review by bziondik — CHANGES_REQUESTED** [review]

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by AnastasiaArcadia — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**